src/components/presentation/PreLoader.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 
 const PreLoader = () => (
-  <div className="preloader-wrapper big active">
+  <div className="preloader-wrapper big active" style={{ position: 'absolute', left: '50%' }}>
```

Comment:
Please, I wouldn't want you adding inline styles to components. Doesn't this have a style file?

Reply:
Ok @omokaro-faith, will fix that right away